docs/applications/containers/kubernetes/static-site-linode-kubernetes-engine/index.md

```diff
@@ -544,24 +609,32 @@ kubernetes&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Clus
 static-site-service&nbsp;&nbsp;LoadBalancer&nbsp;&nbsp;10.128.99.240&nbsp;&nbsp;192.0.2.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80:32648/TCP&nbsp;&nbsp;&nbsp;100m
 {{</ output >}}
 
-    Note the external IP address of the Service you created. This is the IP address of the NodeBalancer, and you can use it to view your static site. In the above example, the IP address is `192.0.2.1`. Navigate to the external IP address in the browser of your choice to view your static site.
+1.  Note the external IP address of the Service you created. This is the IP address of the NodeBalancer, and you can use it to view your static site.
+
+1.  In the above example, the IP address is `192.0.2.1`. Navigate to the external IP address in the browser of your choice to view your static site. You should see the same content as when you tested your Docker Image on your workstation.
+
+## Next Steps
+
+If you'd like to continue using the static site that you created in this guide, you may want to assign a domain to it. Review the [DNS Records: An Introduction](/docs/networking/dns/dns-records-an-introduction/) and [DNS Manager](/docs/platform/manager/dns-manager/) guides for help with setting up DNS. When setting up your DNS record, use the external IP address that you noted at the end of the previous section.
+
+If you would rather not continue using the cluster you just created, review the [tear-down section](#tear-down-your-lke-cluster-and-nodebalancer) to remove the billable Linode resources that were generated.
 
-## Tear Down your LKE Cluster and NodeBalancer.
+## Tear Down your LKE Cluster and NodeBalancer
 
-To remove the NodeBalancer you created, all you need to do is delete the underlying Service:
+- To remove the NodeBalancer you created, all you need to do is delete the underlying Service. From your workstation:
```

Comment:
Splitting this section into unordered list of three items:

- delete NB
- delete cluster
- delete kube context